packages/components/src/utils/log/warnIfDeprecatedPropIsUsed.ts

```diff
@@ -0,0 +1,8 @@
+import { getDeprecatedPropWarningMessage } from './helper';
+
+// TODO: add missing unit test
+export const warnIfDeprecatedPropIsUsed = <T extends string>(host: HTMLElement, prop: string, propValue: T): void => {
```

Comment:
Generic and 3rd parameter can be removed since it is possible to get the value via 1st and 2nd parameter identical to how it is done in `warnIfDeprecatedPropValueIsUsed`.

Reply:
Fixed it